frontend/src/app/components/about/about.component.html

```diff
@@ -1,290 +1,204 @@
     </a>
+  </div>
 
-    <br><br>
-    <br><br>
-
+  <div class="enterprise-sponsor">
     <h3 i18n="about.sponsors.enterprise.withRocket">Enterprise Sponsors 🚀</h3>
+    <div class="wrapper">
+      <a href="https://squarecrypto.org/" target="_blank" title="Square Crypto">
+        <img class="image" src="/resources/profile/sqcrypto.svg" />
+        <span>Square</span>
+      </a>
+      <a href="https://gemini.com/" target="_blank" title="Gemini">
+        <img class="image" src="/resources/profile/gemini.svg" />
+        <span>Gemini</span>
+      </a>
+      <a href="https://exodus.com/" target="_blank" title="Exodus">
+        <img class="image" src="/resources/profile/exodus.svg" />
+        <span>Exodus</span>
+      </a>
+    </div>
+  </div>
 
-    <a href="https://squarecrypto.org/" target="_blank">
-      <div class="profile_photo enterprise_sponsor d-inline-block" title="Square Crypto">
-        <img class="profile_img" src="/resources/profile/sqcrypto.svg" />
-        Square
-      </div>
-    </a>
-
-    <a href="https://gemini.com/" target="_blank">
-      <div class="profile_photo enterprise_sponsor d-inline-block" title="Gemini">
-        <img class="profile_img" src="/resources/profile/gemini.svg" />
-        Gemini
-      </div>
-    </a>
-
-    <a href="https://exodus.com/" target="_blank">
-      <div class="profile_photo enterprise_sponsor d-inline-block" title="Exodus">
-        <img class="profile_img" src="/resources/profile/exodus.svg" />
-        Exodus
-      </div>
-    </a>
-
-    <br><br>
-    <br><br>
-
+  <div class="community-sponsor">
     <h3 i18n="about.sponsors.withHeart">Community Sponsors ❤️</h3>
 
-    <ng-container *ngIf="sponsors$ | async as sponsors; else loadingSponsors">
-      <ng-template ngFor let-sponsor [ngForOf]="sponsors">
-        <a [href]="'https://twitter.com/' + sponsor.handle" target="_blank" rel="sponsored">
-          <div class="profile_photo community_sponsor d-inline-block" [title]="sponsor.handle">
-            <img class="profile_img" [src]="'/api/v1/donations/images/' + sponsor.handle" />
-          </div>
-        </a>
-      </ng-template>
-    </ng-container>
-    <br><br>
-    
+    <div class="wrapper">
+      <ng-container *ngIf="sponsors$ | async as sponsors; else loadingSponsors">
+        <ng-template ngFor let-sponsor [ngForOf]="sponsors">
+          <a [href]="'https://twitter.com/' + sponsor.handle" target="_blank" rel="sponsored" [title]="sponsor.handle">
+            <img class="image" [src]="sponsor.imageUrl" />
```

Comment:
This is not correct and introduces a bug

Reply:
reverted:
`<img class="image" [src]="sponsor.imageUrl" />`
to:
`<img class="image" [src]="'/api/v1/donations/images/' + sponsor.handle" />`